{"name": "新增用户", "status": "passed", "attachments": [{"name": "log", "source": "c523fa2c-ba83-4af1-b33c-f63af4d305e9-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "inData", "value": "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"}, {"name": "exp", "value": "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"}], "start": 1605080496245, "stop": 1605080496374, "uuid": "ce08bb3d-dd63-421a-8bb3-4b97440c728f", "historyId": "c1976a9a8969e78f1e4c0584a94dcff6", "testCaseId": "dc3f5c27b8f716b793706de44b0c3c3f", "fullName": "test_UserManage.TestUserManage#test_add_user", "labels": [{"name": "suite", "value": "test_UserManage"}, {"name": "subSuite", "value": "TestUserManage"}, {"name": "host", "value": "dadaxPC008"}, {"name": "thread", "value": "1252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_UserManage"}]}